model_signing/model.py

```diff
@@ -37,7 +38,7 @@
 from typing import Optional
 from serialize import Serializer
 import psutil
-
+import sys
```

Comment:
```suggestion
import sys

```

need 2 blank lines here to satisfy linker